src/main/java/com/launchdarkly/client/StreamProcessor.java

```diff
@@ -171,4 +195,21 @@ int version() {
     }
 
   }
+
+  private final class HeartbeatDetector implements Runnable {
+
+    @Override
+    public void run() {
+      DateTime fiveMinutesAgo = DateTime.now().minusSeconds(DEAD_CONNECTION_INTERVAL_SECONDS);
```

Comment:
Let's rename `fiveMinutesAgo` to `reconnectThresholdTime` or something else not tied to the value of the constant?